Frontend UI stock discovery category name updates

BearachB opened the pull request:
# Changes
1. Added all sectors to Discovery page (each sector displays the top 20 positive movers over the past 24h period).
2. Added a hierarchy of titles (i.e. Top movers -> Top Gainers | Top Losers. Top Stocks By Sector -> Top Industrials | Top Technology)
3. Renamed almost all of the categories to snappier names.
4. Added icons to each category using MaterialUI.
5. Added info buttons to most categories.
6. Added a grey bottom border between each major section.

There is probably going to need to be a bit of discussion on this as we go as what everyone likes is going to be pretty subjective. There's some aspects of this page that I'm still undecided on, however I think it's visually better than the last iteration.

# Screenshots
## Recommended section
![image](https://user-images.githubusercontent.com/78538312/205967620-dd850853-430e-4fed-98f8-4143f42ee813.png)
## Top ESG
![image](https://user-images.githubusercontent.com/78538312/205967701-0336a207-3f6f-4877-82fb-0211e0d33adc.png)
## Info Button
![image](https://user-images.githubusercontent.com/78538312/205967768-46e7ecdb-5cb1-48df-9780-d19431ad1183.png)
## Footer
![image](https://user-images.githubusercontent.com/78538312/205967815-98691413-0e8c-44cf-84bb-7029cebf570b.png)



## Screenshot of React Console
![image](https://user-images.githubusercontent.com/78538312/205967524-6a764739-6a53-418b-85af-ce545786237d.png)

WarrenK-design commented on src/components/stockDiscoveryComponents/stockSummary/stockSummary.js:
```diff
@@ -6,7 +6,29 @@ import MessageAlert from '../../widgets/MessageAlert/MessageAlert';
+import ElectricalServicesIcon from '@mui/icons-material/ElectricalServices';
+import CloudIcon from '@mui/icons-material/Cloud';
+import ScienceIcon from '@mui/icons-material/Science';
+import ElectricBoltIcon from '@mui/icons-material/ElectricBolt';
```
For these not massive but you can destructure the imports and get away with these in one line. 

```
import { IconName1, IconName2, IconName3, IconeName4 } from '@mui/icons-material';
```

BearachB replied:
Good spot, addressed this locally, it'll be in the next push.

WarrenK-design reviewed (dismissed):
Hi Bearach, Good work just a few things. 

# Footer 
When the page is loading it looks really odd because the footer moves all the way up the page.
I'd be in favour of just removing  it all together at this stage not worth spending time on to fix.
![image](https://user-images.githubusercontent.com/61060096/206429280-d99d98cc-abe2-48bf-891b-09e706407b72.png)

# Line breaks 
I think these two clash, took me a min to get it. 
![image](https://user-images.githubusercontent.com/61060096/206429765-de3d40af-b838-4a2c-bde2-7081affe6a9b.png)

When I go to mobile then I taught it was actually the scroll bar confused me a bit.  
![image](https://user-images.githubusercontent.com/61060096/206429930-1fdc2393-c503-49e6-a257-5ec9dcb26127.png)

# Explanations 
This only a small one but first line of these is bit repetitive and I know there's title but takes bitta work to see what they actually ranked on, a fix could just be to bold/colour what is being measured in the text maybe? Green -> Positive , Red -> Negative and bold for others maybe? 
![image](https://user-images.githubusercontent.com/61060096/206430624-9fd0fa69-13c2-42af-a20a-1f00ae41fd29.png)
![image](https://user-images.githubusercontent.com/61060096/206430688-7e5a4e98-6d5d-45b4-9955-9e01809a8062.png)
![image](https://user-images.githubusercontent.com/61060096/206430788-40d4066f-1581-429f-a520-6adc6aede64d.png)
![image](https://user-images.githubusercontent.com/61060096/206430835-a39239d2-b01d-44c1-abbf-db7d520e9f70.png)
![image](https://user-images.githubusercontent.com/61060096/206431019-a3bb576f-126a-43bc-ad34-f6e780a5e307.png)
![image](https://user-images.githubusercontent.com/61060096/206431079-cfdf117f-3f55-4a20-b66d-2b686fced04a.png)

joeycor92 commented:
Looking good. First thing I noticed on desktop though is that it's not clear that there's more down further.
<img width="600" alt="image" src="https://user-images.githubusercontent.com/81182125/206431713-6b18fd8d-02e3-4581-a828-5cb319e25f1f.png">

Maybe reduce the spacing. There's a lot of whitespace there on the page. 

Agreed about the footer, noticed that when the page was loading.

Just on the wording, I can see what Warren means about the repetition, it keeps saying "This section shows the top 20...". Just wondering if there's a way around it. Do we need to say top 20?

beckskelly commented:
Hey Bearach, looking good, I'd get rid of those horizontal lines and remove the spacing like the others have said, but I like this overall!

WarrenK-design commented:
> Looking good. First thing I noticed on desktop though is that it's not clear that there's more down further. <img alt="image" width="600" src="https://user-images.githubusercontent.com/81182125/206431713-6b18fd8d-02e3-4581-a828-5cb319e25f1f.png">
> 
> Maybe reduce the spacing. There's a lot of whitespace there on the page.

I think adding scroll bar on website would fix this just to show it's scrollable, people who maybe cant use mouse wheel and only click would be able to use website too.

BearachB commented:
Thanks for the comments @WarrenK-design, @joeycor92, @beckskelly!

I think I've addressed all the above comments in this latest push, let me know if there's anything else you want me to cover:
- Footer now loads in with the stock summary so it isn't hanging there on its own while the rest is loading.
- Section borders have been removed.
- Blue background added to the section titles.
- Text in the info buttons has been coloured for positive/negative and bolded in other places.
- @beckskelly said she would add in the scrollbar in her branch she's on now. It would take too long to explain how to do it to me so she said it would be easier overall to do it on hers.

# Screenshots
## No Footer Until Load
![image](https://user-images.githubusercontent.com/78538312/206489109-5946b9ce-a25a-49c6-a1c7-6c30bfe8b6fa.png)

## Blue Background (Also with white info buttons, and no more section borders)
![image](https://user-images.githubusercontent.com/78538312/206488969-47e73765-647b-4a74-a324-2bf8d48730e7.png)

## Info Button Coloured Text:
![image](https://user-images.githubusercontent.com/78538312/206489523-29009d00-72ac-495a-9792-86ce402fe5fd.png)
![image](https://user-images.githubusercontent.com/78538312/206489594-906fb63a-38bc-4f68-acca-b6b0adc156d6.png)
![image](https://user-images.githubusercontent.com/78538312/206489727-493363fd-029b-48bd-9255-c65ce5fa9c94.png)

joeycor92 reviewed (commented):
Lot of serious work done here. Can definitely see the benefit of all the changes and really like the updates to the information modals. 

The one thing for me is that the banners and titles all seem very big on desktop and they are taking up a lot of space on the screen. Should we consider making the titles smaller and decreasing the margins/padding? 

Happy to approve if everyone else is happy with it.

WarrenK-design commented:
Hi Bearach, 

It looks good but as discussed feel its a bit busy and the search bar gets alittle lost within it. 
Sorta looks like each section is a mini footer along the page. 
![image](https://user-images.githubusercontent.com/61060096/206563266-14e84c0d-543a-4a68-9e82-cf78727de38f.png)
Dont mind if change to a plainer solution like below or @beckskelly had a solution maybe some underline if we really wanted?
![image](https://user-images.githubusercontent.com/61060096/206563430-dc6369fd-ed70-4d25-8439-747232b38130.png)
For these then too just small thing, think we can say top once in section heading then remove it from each of these? 
![image](https://user-images.githubusercontent.com/61060096/206563566-ef8cf22c-315e-4ba1-a824-28c7e18294e7.png)

BearachB commented:
OK @WarrenK-design addressed your two changes in the latest push.
- Removed "Top" from sector categories.
- Removed blue banner from behind section headings.  

Still not fully sold on the lack of colour, we really could do with some form of background images to break it up however I don't exactly know how to go about that. Still looking very serious.

beckskelly reviewed (approved):
Happy with it for now, I'll have a think over the weekend about making the page a bit more fun, its low on fun and colour for now
